rasa/shared/core/training_data/visualization.py

```diff
@@ -517,7 +570,9 @@ def visualize_neighborhood(
 
     if should_merge_nodes:
         _merge_equivalent_nodes(graph, max_history)
-    _replace_edge_labels_with_nodes(graph, next_node_idx, nlu_training_data)
+    _replace_edge_labels_with_nodes(
+        graph, next_node_idx, None
+    )  # None instead of nlu_training_data)
```

Comment:
If you pass nlu_training_data the intent name will be replaced by some random user example. The diagram becomes mostly unreadable with that. If you have an idea how to make that behavior parametrizable be my guest.

Reply:
We need an improved comment describing concisely why None is required here (as explained above).

Reply:
@ancalita @raoulvm - I think you have two usecases here, 
If you are passing nlu_training_data, it takes a random example from the training set, which i believe is fine. I wouldn't pass None here because then userMessage is never shown. There is already a check in the method to see when nlu_training_data is None
If you only want to see the intents, then don't pass the nlu training data, it is already an optional param in the cli.